index.js

```diff
@@ -217,6 +217,11 @@ function fromJsonObject (that, object) {
 if (Buffer.TYPED_ARRAY_SUPPORT) {
   Buffer.prototype.__proto__ = Uint8Array.prototype
   Buffer.__proto__ = Uint8Array
+  if (typeof global.Symbol !== 'undefined' && Symbol.species &&
```

Comment:
Sorry for the delay; of course it does. Pushed a new version which is worded that way. What do you think?